### DISCUSSION TITLE
need access to GitHub copilot.

### DISCUSSION THREAD

**Discussion by EdenBendheim** [community/community#142850] [GitHub Education]
### Hi everyone,

I have been verified yesterday but this is really the only feature I need working. Checking if anyone on here can help other than just waiting. 
Thanks,
Eden

**Comment by davevad93** [comment]
Hi @EdenBendheim ,

GitHub no longer provides manual reviews, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @EdenBendheim was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Leokratis** [reply]
The FAQs say that it can take about 3 days to get your perks delivered to you. It also mentions that in periods like the start of an academic new year it can take 5 or more days depending on the number of requests they have.

**Reply by EdenBendheim** [reply]
Yes it has been almost 5 days since I originally applied, and I still have not received anything :(.

**Reply by davevad93** [reply]
Hey @EdenBendheim , I noticed that you now have a `PRO` badge, which means you should already have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Comment by peachmontt** [comment]
Log Out and Back In: Sometimes, simply logging out of your GitHub account and then logging back in can help refresh the connection.

Check Permissions: Ensure that GitHub Copilot has the necessary permissions in your GitHub account settings. You can check this under "Settings" > "Applications" > "Authorized OAuth Apps" and make sure Copilot is listed and authorized.

Update Your IDE: Make sure that your IDE (e.g., Visual Studio Code) is up to date, along with the GitHub Copilot extension. Sometimes, updates can resolve bugs and improve functionality.

Check for Outages: Visit the [GitHub Status page](https://www.githubstatus.com/) to see if there are any ongoing issues with GitHub Copilot or related services.
